client/client-multi/client-c/src/kaa/platform-impl/common/common_key_utils.c

```diff
@@ -369,10 +272,10 @@ void ext_get_sha1_public(uint8_t **sha1, size_t *length)
     *length = KAA_SHA1_PUB_LEN;
 }
 
-void ext_get_sha1_base64_public(uint8_t **sha1, size_t *length)
+kaa_error_t ext_get_sha1_base64_public(uint8_t **sha1, size_t *length)
 {
     if (!sha1 || !length) {
-        return;
+        return KAA_ERR_BADPARAM;
```

Comment:
This function should have one more return statement, but it wasn't updated to match new signature.